src/opnsense/mvc/app/library/OPNsense/Routing/Gateways.php

```diff
@@ -267,9 +281,11 @@ public function getGateways()
             foreach ($dynamic_gw as $intfgws) {
                 foreach ($intfgws as $gw_arr) {
                     if (!empty($gw_arr)) {
-                        $gw_arr['disabled'] = true;
-                        $gw_arr['defunct'] = true;
-                        unset($gw_arr['gateway']);
+                        if(!self::IsVPNInterface($gw_arr)){
+                            $gw_arr['disabled'] = true;
+                            $gw_arr['defunct'] = true;
+                            unset($gw_arr['gateway']);
+                        }
```

Comment:
In some use cases you may want a VPN interface to be your default gateway, so the IsVPNInterface() check here should probably be removed to maintain functional consistency.

Reply:
@drivera73 this PR won't be included, we're currently opting for this  https://github.com/opnsense/core/commit/dba70c0ead9b55e4be8ac37dad5c2afbfe3209e8